software/controller/lib/hal/adc.cpp

```diff
@@ -89,10 +89,20 @@ static constexpr int MaxAdcReading = (OversampleLog2 >= 4) ? 65536 : (1 << (12 +
 // Set sample time ([RM] 16.4.12). I'm using 92.5 A/D clocks to sample.
 // A/D sample time in CPU clock cycles.  Fixed for now
 // This is the time we give the analog input to charge the A/D sampling cap.
-static constexpr int AdcSampleTime = 92;
+enum class ADCSampleTime {
+  s2_5 = 2,
```

Comment:
so, `2_5` = 2.5, etc..? Maybe explain this convention in comments? Why do we need a truncated number representation of this? Only for the `AdcConversionTime` as a sort of estimate?

Reply:
This will change due to your last comment, but I will try to make it clearer